Automattic/jetpack
# VideoPress: hide VideoPress block and update Video block description

Fixes Automattic/greenhouse#990
Fixes #11480

#### Changes proposed in this Pull Request:

The VideoPress embed block is confusing when using VideoPress.
This PR hides the VideoPress embed block in favor of the Video block (where we could still embed a VideoPress URL there).
It also updates the video block example and description to be more VideoPress-specific when VideoPress is enabled.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* On a Jetpack site with VideoPress enabled
* Create a new post or edit an existing one
* Type /videopress
* ✅ The video block should appear and the VideoPress embed block should not be visible

![Capture d’écran 2021-11-15 à 12 06 41](https://user-images.githubusercontent.com/1420594/141771622-10a07470-838b-4469-bad1-337bb1cfd302.png)

* Clear your line and click on the "+" button 
* Search for "videopress"
* ✅ The video block should appear and the VideoPress embed block should not be visible

![Capture d’écran 2021-11-15 à 12 07 39](https://user-images.githubusercontent.com/1420594/141771788-c51ccec8-18ff-4758-a8e1-dc51adc421cb.png)

* Click on "Browse all"
* ✅ The VideoPress embed block should not be visible in this list
* ✅ The Video block should have an updated example and description

![Capture d’écran 2021-11-15 à 12 09 20](https://user-images.githubusercontent.com/1420594/141771981-e18c289e-154b-45dd-9bc7-f1ce1be0be01.png)

* Add a video block
* Click on "Insert from URL"
* Insert a VideoPress link ( you can try https://videopress.com/v/Ygmx4akX )
* ✅ The block should be transformed to a VideoPress embed

![Capture d’écran 2021-11-15 à 12 11 03](https://user-images.githubusercontent.com/1420594/141772205-5054ac0f-97a9-4a44-9087-d50b4a262959.png)

* In a new tab, go to your Jetpack settings and disable VideoPress
* Refresh your edited post 
* ✅ The VideoPress embed block should be available again and the video block description should be the default core video description (no mentions of VideoPress)
